src/api/messages.ts

```diff
@@ -1,25 +1,59 @@
 import { extensionPort } from "src";
 
-export const showConfirm = async (str: string) => {
-  return extensionPort.showConfirm(str);
+/**
+ * Shows a confirmation toast message within the Replit workspace for `length` milliseconds.  Returns the ID of the message as a UUID
+ */
+export const showConfirm = async (
+  str: string,
+  length: number = 4000,
+  style?: React.CSSProperties
```

Comment:
i'm not sure style props work through the message boundary. and i'm not sure they even should be a part of the API